ui_components/src/village/WarTable.js

```diff
@@ -208,6 +429,66 @@ export function WarTable({
                     </div>
                 </div>
             </div>
+            <div className="row third" style={{display: "none"}}>
+                <div className="column first">
+                    <svg height="0" width="0">
+                        <defs>
+                            <filter id="war_record_hover">
+                                <feGaussianBlur in="SourceAlpha" stdDeviation="2" result="blur" />
+                                <feFlood floodColor="white" result="floodColor" />
+                                <feComponentTransfer in="blur" result="opacityAdjustedBlur">
+                                    <feFuncA type="linear" slope="1" />
+                                </feComponentTransfer>
+                                <feComposite in="floodColor" in2="opacityAdjustedBlur" operator="in" result="coloredBlur" />
+                                <feMerge>
+                                    <feMergeNode in="coloredBlur" />
+                                    <feMergeNode in="SourceGraphic" />
+                                </feMerge>
+                            </filter>
+                        </defs>
+                    </svg>
```

Comment:
This looks like a good candidate to be its own component

Reply:
Yep makes sense, we only really care about the color, slope, and deviation for the sake of how the output looks so those can be a parameter for a `GlowUtil` or similarly named utility.